components/financial_assistance/app/views/financial_assistance/applications/_non_oe_year_selection_form.html.erb

```diff
@@ -11,7 +11,7 @@
           <% previous_year = calendar_year - 1 %>
           <h2 class="darkblue"><%= l10n("faa.year_selection_header") %></h2>
           <p><%= l10n("faa.year_selection_subheader") %></p>
-          <p><b><%= calendar_year %></b><br> <%= l10n("faa.non_oe_see_if_you_qualify_1") %>.<br><%= link_to l10n("Learn_more_about_life_changes").humanize, FinancialAssistanceRegistry[:iap_year_selection_form].setting(:iap_year_sep_link).item %>.</p>
+          <p><b><%= calendar_year %></b><br> <%= l10n("faa.non_oe_see_if_you_qualify_1", year: calendar_year) %>.<br><%= sanitize link_to l10n("Learn_more_about_Special_Enrollment_Periods").humanize, FinancialAssistanceRegistry[:iap_year_selection_form].setting(:iap_year_sep_link).item %>.</p>
```

Comment:
I think the `l10n("Learn_more_about_Special_Enrollment_Periods")` chunk should probably be added as a new entry in the translation file. It will definitely work how you have it but generally is anything over a word or two probably should in the translation file. (We abuse this all over the code base but for the sake of 'boy scouting' we might as well take the time to improve it here.)

Reply:
Sounds good! I will change that

Reply:
Pushed commit with the update!